README.md

```diff
@@ -141,6 +141,62 @@ const searchResult = await shopperSearch.productSearch({
 
 Invalid query parameters that are not a part of the API and do not follow the `c_` custom query parameter convention will be filtered from the request and a warning will be displayed.
 
+### Custom APIs
+
+The SDK supports calling custom APIs with a helper function, `callCustomEndpoint`.
+
+Example usage:
+
+```javascript
+import pkg from 'commerce-sdk-isomorphic';
+const { helpers } = pkg;
+
+const CLIENT_ID = "<your-client-id>";
+const ORG_ID = "<your-org-id>";
+const SHORT_CODE = "<your-short-code>";
+const SITE_ID = "<your-site-id>";
+
+// client configuration parameters
+const clientConfig = {
+  parameters: {
+    clientId: CLIENT_ID,
+    organizationId: ORG_ID,
+    shortCode: SHORT_CODE,
+    siteId: SITE_ID,
+    // Custom API path parameters
+    endpointPath: 'customers',
+    apiName: 'loyalty-info',
+    apiVersion: 'v1', // defaults to v1 if not provided
+  },
+};
+
+// Flag to retrieve raw response or data from helper function
+const rawResponse = false;
+const accessToken = '<INSERT ACCESS TOKEN HERE>';
+
+let response = await helpers.callCustomEndpoint(
```

Comment:
I don't feel strongly, but would you consider an alternative argument structure for `callCustomEndpoint`?

```js
await helpers.callCustomEndpoint({
  options: myOptions,
  clientConfig: myClientConfig,
  rawResponse: true
})
```

This interface makes it much more clear to folks using regular old JavaScript what the impact of the three arguments will be.

It would also allow you to significantly shrink this code example:

```js
await helpers.callCustomEndpoint({
  options: {
    method: "GET",
    parameters: {
      queryParameter: "queryParameter1",
      siteId: SITE_ID,
    },
    headers: {
      "Content-Type": "application/json",
      authorization: `Bearer <your-access-token>`,
    },
  },
  clientConfig: {
    clientId: "<your-client-id>",
    organizationId: "<your-org-id>",
    shortCode: "<your-short-code>",
    siteId: "<your-site-id>",
    // Custom API path parameters
    endpointPath: "customers",
    apiName: "loyalty-info",
    apiVersion: "v1", // defaults to v1 if not provided
  },
  rawResponse: true,
});
```

Reply:
Would you also consider showing an example of a mutation request eg. a POST where you pass a request body?